config.yaml

```diff
@@ -48,6 +48,11 @@ mysql_config_version: 0
 # mysql_key_value_table_name:
 
 
+# (OPTIONAL | Default: "session_access_token_signing_keys") string value. Specify the name of the table that will store 
```

Comment:
Oh we don't need these config values for new tables. They were only for older tables.